components/Slider/src/MDCSlider.m

```diff
@@ -31,7 +31,9 @@
   return MDCPalette.bluePalette.tint500;
 }
 
+API_AVAILABLE(ios(10.0))
```

Comment:
Please remove the API_AVAILABLE from the interface declaration as there may be additional things added in the future. Also, I believe you can just set the property to API_AVAILABLE and it does the trick